src/api/app/views/notifications/comment_for_request.text.erb

```diff
@@ -1,6 +1,7 @@
 <%- comment = notification.notifiable %>
-Visit <%= url_for(controller: 'webui/request', action: :show, number: comment.commentable.number, host: @host, only_path: false) %>
+<%- request_number = comment.commentable.is_a?(BsRequest) ? comment.commentable.number : comment.commentable.bs_request.number %>
```

Comment:
As we use duck typing, this should use `respond_to?(:number)` instead of discriminating by class. A better way would be `comment.commentable` returning the right object...

Reply:
Applied.

This change should be followed by a refactoring, most likely using decorators.